### ISSUE TITLE
ZAutomation API authentication with the find.z-wave.me service as an intermediary

### ISSUE THREAD

**Issue by pathec** [Z-Wave-Me/home-automation#411]
Hi,

I'm working on a third party client and try to connect Z-Way via the remote service.

I've tried the POST request 'https://find.z-wave.me:443/zboxweb?act=login&login=***/admin&pass=***'. But the ZBW-SESSID header is empty (HTTP response status is 200 and the content is the find.z-wave.me/zboxweb HTML-Code). My credentials work through the web interface.

In the forum I found only that: https://forum.z-wave.me/viewtopic.php?f=3422&t=21460&start=10&hilit=proxy.

Is there another way?

**Comment by PoltoS** [comment]
Make sure act=login&login=***/admin&pass=*** is passed via POST data and not in URL.

Here is a Postman example
![find postman](https://cloud.githubusercontent.com/assets/3664383/21624302/8bc77560-d217-11e6-9826-3c48fdefcd50.png)

**Comment by pathec** [comment]
Thank you very much! It works when pass the data via POST data.

**Comment by pathec** [comment]
Hi @PoltoS,

I've another question.

How can a WebSocket connection to Z-Way be made using the remote service. I've tried the following: 
First I loaded the cookies (ZWAYSession, ZBW_SESSID) using HTTPS (as described above). Then I connect to Z-Way with: `wss://find.z-wave.me:443` and the stored cookies. The WebSocket connection event is triggered, but then a error with the message: _Didn't switch protocols_ occurres, which indicates that the server doesn't support WebSockets. The same code works for local access with: _ws://LOCAL_IP:8083_.

Any idea?
